README.md

```diff
@@ -31,30 +31,30 @@ configuration for each of your entities namespaces:
 
 ```php
 <?php
-return array(
-    'doctrine' => array(
-        'driver' => array(
+return [
```

Comment:
@Maaslos-UG 

All of these coding style fixes have nothing to do with making this project compatible with ZF3.

Can we apply coding style fixes in a separate PR, one that addresses coding style fixes only?